index.bs

```diff
@@ -1892,8 +1899,8 @@ usage.
   <tr>
     <td>{{UnknownError}}</td>
     <td>
-      The operation failed for reasons unrelated to the database
-      itself and not covered by any other errors.
+      The operation failed for transient reasons unrelated to the
+      database itself and not covered by any other error.
```

Comment:
In my opinion, this text scopes `UnknownError` a bit too much. If it's any other type of error, a developer should be able to look at the spec and deduce exactly what the cause was; `UnknownError` is basically for anything unspecified.

Maybe "or not covered" instead of "and not covered" ?

Reply:
Done